src/main/java/gov/cabinetoffice/gapfindapiadmin/repositories/GrantAdminRepository.java

```diff
@@ -0,0 +1,9 @@
+package gov.cabinetoffice.gapfindapiadmin.repositories;
+
+import gov.cabinetoffice.gapfindapiadmin.models.GrantAdmin;
+import org.springframework.data.repository.CrudRepository;
+
+public interface GrantAdminRepository extends CrudRepository<GrantAdmin, Integer> {
+    GrantAdmin findByGapUserUserSub(String userSub);
```

Comment:
Some of the repositories are returning optional objects and some are returning concrete objects for the same kinds of method calls. Is there a reason for that? Could we decide on a consistent approach and stick to it?

Reply:
removed the only optional we had, and added a null check